app/migrations/Version20170419153243.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20170419153243 extends AbstractMigration
```

Comment:
The version number will have to be incremented to the current date to avoid problems with more recent migration version numbers.